pkgs/development/libraries/libhwy/default.nix

```diff
@@ -10,6 +10,14 @@ stdenv.mkDerivation rec {
     rev = version;
     hash = "sha256-Gym2iHq5ws9kuG4HWSQndD8hVugV4USZt6dUFnEkLwY=";
   };
+  patches = [
```

Comment:
LGTM

I suggest adding a small comment here referring upstream issue: https://github.com/google/highway/issues/1613 . That way if should be easier to evaluate if the patch is worth keeping on next update.

Another option is to try to pull in forward-fix (committed upstream a few hours ago): https://github.com/google/highway/commit/7ad89efa911cb906ccf3f78fe510db415e921801 . Don't know if it applies cleanly and passes the tests. Might not,